workshops/views.py

```diff
@@ -120,7 +123,7 @@ def get_context_data(self, **kwargs):
         else:
             context['title'] = 'New object'
 
-        context['form_helper'] = bootstrap_helper
+        context['form_helper'] = BootstrapHelper()
```

Comment:
The helper can be edited by views (i.e. by `TrainingProgressCreate`). We should have one new helper for each view, not one shared for all views.